recipes/utfcpp/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conans import ConanFile, tools
+
+
+class UtfCppConan(ConanFile):
+    name = "utfcpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nemtrif/utfcpp"
+    description = "UTF-8 with C++ in a Portable Way"
```

Comment:
Either include the settings as input for Conan in this recipe or remove the package_id() not needed for a recipe without settings (header only by default)
```suggestion
    description = "UTF-8 with C++ in a Portable Way"
    settings = "os", "compiler", "arch", "build_type"
```

Reply:
Fixded in latest commit